pkg/virt-v2v/config/variables.go

```diff
@@ -160,6 +164,11 @@ func (s *AppConfig) validate() error {
 			}
 			if s.SecretKey == "" {
 				return s.envMissingError(SecretKey)
+			}
+			if s.VirtIoWinLegacyDrivers != "" {
+				if _, err := os.Stat(s.VirtIoWinLegacyDrivers); os.IsNotExist(err) {
```

Comment:
Have you tested the virt-v2v with the `VIRTIO_WIN` pointing to the wrong path/empty dir?
I worry that the virt-v2v will fail so we would need to unset the `VIRTIO_WIN` or we would need to rename this and pass the environment variable during the virt-v2v call